paraatm/simulation_method/vcas.py

```diff
@@ -145,19 +148,19 @@ def simulation(self, input_cmd=None):
             ignore_index=True)  # end cmd
 
         # self.init_NATS()
-        self.environmentInterface.load_rap(os.environ.get('NATS_HOME') + '/' + 'share/tg/rap')  # default wind file
+        self.environmentInterface.load_rap(os.environ.get('GNATS_HOME') + '/' + 'share/tg/rap')  # default wind file
         self.aircraftInterface.load_aircraft(self.fp_file, self.mfl_file)
         aclist = self.aircraftInterface.getAllAircraftId()
         self.simulationInterface.setupSimulation(self.sim_time, 1)
         self.simulationInterface.start(1)
         # Use a while loop to constantly check server status.
         while True:
             server_runtime_sim_status = self.simulationInterface.get_runtime_sim_status()
-            # print(server_runtime_sim_status)
+            time.sleep(0.1)
             if (server_runtime_sim_status == self.NATS_SIMULATION_STATUS_PAUSE):
                 break
-            else:
-                time.sleep(0.1)
+            elif (server_runtime_sim_status == self.NATS_SIMULATION_STATUS_ENDED):
+                raise Exception('Simulation stopped unexpectedly. Please rerun.')
 
         curr_t = self.starttime + self.simulationInterface.get_curr_sim_time()
         ac = self.aircraftInterface.select_aircraft(aclist[0])
```

Comment:
When I run `vcas_example.py`, I get the following error, which points to this line:
```
Begin flight propagation for 1.000000 seconds duration from 0.000000 to 1000.000000 second with time step 1.0 seconds (surface), 1.0 seconds (terminal area), 1.0 seconds (above TRACON)

    Simulation time:  0 hours
    Duration satisfied.  Automatically pausing at time = 1.000000 seconds
Traceback (most recent call last):
  File "paraatm/tests/vcas_example.py", line 29, in <module>
    track = sim()  # call simulation function using NatsSimulationWrapper
  File "/home/jmcfarland/work/git/para-atm/paraatm/io/gnats.py", line 277, in __call__
    self.simulation(*args, **kwargs)
  File "/home/jmcfarland/work/git/para-atm/paraatm/simulation_method/vcas.py", line 167, in simulation
    ac = self.aircraftInterface.select_aircraft(aclist[0])
  File "/home/jmcfarland/venv/para-atm/lib/python3.6/site-packages/JPype1-0.6.3-py3.6-linux-x86_64.egg/jpype/_jarray.py", line 60, in __getitem__
    return _jpype.getArrayItem(self.__javaobject__, ndx)
jpype._jexception.ArrayIndexOutOfBoundsExceptionPyRaisable: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
JVM has been shutdown
```
Also if I check the length of `aclist`, it is 0.